openmc/universe.py

```diff
@@ -446,6 +472,13 @@ def create_surface_elements(node, element):
             # Call the recursive function from the top node
             create_surface_elements(self.region, xml_element)
 
+        if self.temperature is not None:
+            if isinstance(self.temperature, Iterable):
+                element.set("temperature", ' '.join(
+                            [str(t) for t in self.temperature]))
```

Comment:
I'm sure we're not consistent about this, but it's generally better to use a generator expression, i.e.
```
' '.join(str(t) for t in self.temperature)
```